docs/getting-started/my-account/README.md

```diff
@@ -158,19 +147,19 @@ You can choose between several email reminders.
 
 Default: Enable daily email reminders: 2am, Monday - Friday.
 
-You can choose to receive emails immediately, or only on certain days and times, temporarily pause reminder emails, or opt for no reminders at all. 
+You can choose to receive emails immediately, or only on certain days and times, temporarily pause reminder emails, or opt for no reminders at all.
 
-![Email reminders](EmailNotificationsv12.png)
+ ![email-reminders](Bildschirmfoto 2022-03-02 um 16.02.09.png)
```

Comment:
please rename the image file and change the name here, so there are no spaces in it.